frontend/public/components/monitoring/alert-manager-utils.tsx

```diff
@@ -4,28 +4,39 @@ import { Base64 } from 'js-base64';
 import { safeLoad, safeDump } from 'js-yaml';
 
 import { k8sPatch, K8sResourceKind } from '../../module/k8s';
-import {
-  AlertManagerConfig,
-  AlertManagerReceiver,
-  AlertManagerRoute,
-} from './alert-manager-config';
+import { AlertManagerConfig } from './alert-manager-config';
```

Comment:
nit: all of the `AlertManager` names throughout should really be `Alertmanager`

Reply:
fixed